rspace/src/main/scala/coop/rchain/rspace/IStore.scala

```diff
@@ -126,7 +126,7 @@ trait IStore[F[_], C, P, A, K] {
           value
             .sorted(Ordering.by((tu: TrieUpdate[C, P, A, K]) => tu.count).reverse)
             .headOption match {
```

Comment:
It seems the delete and insert cases can be deduplicated/merged

Reply:
huh... not convinced, but sure.